test/spec/provider/camunda/element-templates/cmd/ChangeElementTemplateHandlerSpec.js

```diff
@@ -787,6 +787,91 @@ describe('element-templates - cmd', function() {
 
     });
 
+    describe('with camunda:Field', function() {
+
+      var diagramXML = require('./task-clean.bpmn');
+
+      var currentTemplate = require('./fieldinjections');
+
+      beforeEach(bootstrapModeler(diagramXML, {
+        container: container,
+        modules: [
+          coreModule,
+          modelingModule,
+          propertiesPanelCommandsModule,
+          elementTemplatesModule
+        ],
+        moddleExtensions: {
+          camunda: camundaModdlePackage
+        }
+      }));
+
+      beforeEach(inject(function(elementRegistry) {
+        var taskShape = elementRegistry.get('Task_1');
+
+        applyTemplate(taskShape, currentTemplate);
+      }));
+
+
+      it('execute', inject(function(elementRegistry) {
+
+        // given
+        var taskShape = elementRegistry.get('Task_1'),
+            task = taskShape.businessObject;
+
+        // when
+        applyTemplate(taskShape, null);
+
+        var fieldInjections = findExtensions(taskShape, [ 'camunda:Field' ]);
+
+        // then
+        expect(task.get('camunda:modelerTemplate')).not.to.exist;
+
+        // removing a task template does
+        // not change the applied values
+        expect(fieldInjections).to.exist;
+        expect(fieldInjections).to.jsonEqual([
+          {
+            $type: 'camunda:Field',
+            string: 'My String Field Injection',
+            name: 'sender'
+          },
+          {
+            $type: 'camunda:Field',
+            string: 'My String Field Injection 2',
+            name: 'sender2'
+          },
+          {
+            $type: 'camunda:Field',
+            expression: '${PerfectExpression}',
+            name: 'sender3'
+          }
+        ]);
+      }));
+
+
+      it('undo', inject(function(elementRegistry, commandStack) {
+
+        // given
+        var taskShape = elementRegistry.get('Task_1'),
+            task = taskShape.businessObject;
+
+        applyTemplate(taskShape, null);
+
+
+        // when
+        commandStack.undo();
+
+        var fieldInjections = findExtensions(taskShape, [ 'camunda:Field' ]);
+
+        // then
+        expect(task.get('camunda:modelerTemplate')).to.eql(currentTemplate.id);
+
+        expect(fieldInjections).to.exist;
+      }));
+
+    });
```

Comment:
Add test case that verify:

* camunda:Field on Connectors
* interaction with existing Camunda Field declarations (will they be kept or deleted)?
* camunda field updating